src/components/ListGroup.js

```diff
@@ -4,11 +4,12 @@ import { Link } from "react-router-dom";
 import Task from './Task';
 import MyButton from './ui/button/MyButton';
 import '../styles/List.css'
+import MyModal from './ui/modal/MyModal';
 
 const ListGroup = ({group, groups, setGroups, removeGroup, addCompleted, addNotCompleted, setLocalGroups}) => {
```

Comment:
1. For more readability change to props:
```suggestion
const ListGroup = (props) => {
```
OR
```suggestion
const ListGroup = ({
  group, 
  groups, 
  setGroups, 
  removeGroup, 
  addCompleted, 
  addNotCompleted, 
  setLocalGroups
}) => {
```